cmd/agent-js-worker/main.go

```diff
@@ -531,6 +545,56 @@ func agentOpts(opts *agentStartOpts) ([]aries.Option, error) {
 	return options, nil
 }
 
+func createStore(opts *agentStartOpts) (storage.Provider, error) {
```

Comment:
I don't see SDS for `agent-rest` and agent-mobile. See the following lines:

https://github.com/trustbloc/agent-sdk/blob/master/cmd/agent-rest/startcmd/start.go#L232
https://github.com/trustbloc/agent-sdk/blob/master/cmd/agent-mobile/pkg/wrappers/command/aries.go#L122

Reply:
For now I've added TODOs for these tasks, since the SDS integration isn't actually complete yet - it will require some changes to aries first (see the other TODO comments I left in this PR). Once the SDS integration is complete then I will add it to aries-rest and aries-mobile. Note that there are other pieces (e.g. the remote KMS and hub-auth) that will also need to be added to aries-rest and aries-mobile eventually.